.github/ISSUE_TEMPLATE/release.md

```diff
@@ -20,9 +20,13 @@ assignees: ''
 
 ## Release checklists
 
+Replace the items in the task lists below with the applicable Pull Requests / Issues
+
 ```[tasklist]
 ### General pre-requisites (before feature freeze)
-- [ ] Bump operator-rs to latest version in all operators. This should be done early in the release cycle to leave sufficient time for testing etc.
+- [ ] [Update and Release operator-rs workspace members](https://github.com/stackabletech/operator-rs/issues/new?assignees=&labels=epic&projects=&template=release-workspace-members.md&title=chore%3A+Update+and+release+workspace+members)
+- [ ] [TODO: Update Rust toolchain](https://github.com/stackabletech/operator-templating)
+- [ ] [Update Container Images](https://github.com/stackabletech/docker-images/issues/new?assignees=&labels=epic&projects=&template=pre-release.md&title=chore%3A+Update+Container+Images+for+Stackable+Release+XX.%28X%29X)
```

Comment:
I think this should be step 1 (as it was other times). Assuming you mean updating all images).

Reply:
I think it should also be a tracking issue, as the product updates tend to happen individually for reduced friction.

```suggestion
- [ ] [Update Container Images](https://github.com/stackabletech/docker-images/issues/new?assignees=&labels=epic&projects=&template=pre-release.md&title=chore%28tracking%29%3A+Update+Container+Images+for+Stackable+Release+XX.%28X%29X)
```

_Unfortunately I can't move it up the list here, so that would be a separate change._

Reply:
Oh actually, this list was updated in https://github.com/stackabletech/issues/pull/602

So we can either merge that one now, then update the links to other issue templates later, or merge it into this change (though it also serves as a tracking issue)?

Reply:
Now that I think about it, this commit could just be applied on top of #602. Is that ok?

Reply:
> I think this should be step 1 (as it was other times). Assuming you mean updating all images).

I feel like we should first ensure that our operators run fine with the newest Rust deps and on the latest Rust toolchain before we bump the product images.

> I think it should also be a tracking issue, as the product updates tend to happen individually for reduced friction.

Yes, you are right. That title query parameter is however generated by the issue template here: https://github.com/stackabletech/docker-images/blob/main/.github/ISSUE_TEMPLATE/pre-release.md. So I would like to prefer updating the template, before we use the updated link.

> Now that I think about it, this commit could just be applied on top of https://github.com/stackabletech/issues/pull/602. Is that ok?

There is only one commit in this PR. I can merge this branch onto the other one, closing this one. I can then continue my work on the other branch.

Reply:
Changes removed here, added on top of #602